src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+export const checkStatus = () => ({
+  type: 'CHECKSTATUS',
+});
```

Comment:
- [ ] As mentioned above it would be better to extract your action types in a file that will export them
- [ ] Secondly, (OPTIONAL since you are using redux toolkit) you should do the same for your actions (in this case `checkStatus`)